packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -271,8 +271,9 @@ contract Initializer_Test is Bridge_Initializer {
     ///         3. The `initialize()` function of each contract cannot be called more than once.
     function test_cannotReinitialize_succeeds() public {
         // Ensure that all L1, L2 `Initializable` contracts are accounted for, in addition to
-        // OptimismMintableERC20FactoryImpl and OptimismMintableERC20FactoryProxy
-        assertEq(_getNumInitializable() + 2, contracts.length);
+        // OptimismMintableERC20FactoryImpl and OptimismMintableERC20FactoryProxy.
+        // TODO: Add `OptimismPortal2` and `OptimismPortal2Proxy` once the deployment scripts are updated.
```

Comment:
We probably actually don't want an `OptimismPortal2Proxy` and would instead want to just reuse the `OptimismPortalProxy`. Otherwise we have to hook things up in pretty crazy ways. Just need to change behaviors in a few places depending on which implementation is being used.

Reply:
Yeah that's fine. Will leave it up to you when you do the deploy scripts 👍